src/battle_animation.cpp

```diff
@@ -311,6 +315,92 @@ void BattleAnimationBattle::ShakeTargets(int str, int spd, int time) {
 	}
 }
 
+BattleAnimationBattler::BattleAnimationBattler(const lcf::rpg::Animation& anim, std::vector<Game_Battler*> battlers, bool only_sound, int cutoff_frame, bool set_invert) :
+	BattleAnimation(anim, only_sound, cutoff_frame), battlers(std::move(battlers))
+{
+	invert = set_invert;
+}
+
+void BattleAnimationBattler::Draw(Bitmap& dst) {
+	if (IsOnlySound())
+		return;
+	if (animation.scope == lcf::rpg::Animation::Scope_screen) {
+		DrawAt(dst, SCREEN_TARGET_WIDTH / 2, SCREEN_TARGET_HEIGHT / 3);
+		return;
+	}
+
+	for (auto* battler: battlers) {
+		DrawAt(dst, battler->GetDisplayX(), battler->GetDisplayY());
+	}
+}
+
+void BattleAnimationBattler::UpdateScreenFlash() {
+	int r, g, b, p;
+	UpdateFlashGeneric(screen_flash_timing, r, g, b, p);
+	if (r > 0 || g > 0 || b > 0 || p > 0) Main_Data::game_screen->FlashOnce(r, g, b, p, 0);
```

Comment:
This rgba stuff was controversial the last time we had this PR but now it is isolated into Animation2 only so it cannot break other code.
Can you indent this properly? Add {} after the if as in FlashTargets below.

Reply:
Fixed the indentation.